set_up_test_testflight.sh

```diff
@@ -1,12 +1,14 @@
 echo "import('./Tests/Fastfile')" | cat - fastlane/Fastfile | tee fastlane/Fastfile &> /dev/null
 
+readonly CONSTANT_TEAM_ID="<#teamId#>"
```

Comment:
Small question, now `TEAM_ID` is a required field, but before we can use match without `TEAM_ID`?

Reply:
@blyscuit Yes, I think it is because the match uses automatic signing. Now we use update code signing to set code signing manually.

Reply:
@ducbm051291 using match is about setting code signing manually already. In this case, [`Team ID` is used when upgrading project](https://docs.fastlane.tools/actions/update_code_signing_settings/#:~:text=Team%20ID%2C%20is%20used%20when%20upgrading%20project). 

From [the example](https://docs.fastlane.tools/actions/update_code_signing_settings/#:~:text=%23%20manual%20code%20signing%0Aupdate_code_signing_settings(%0A%20%20use_automatic_signing%3A%20false%2C%0A%20%20path%3A%20%22demo%2Dproject/demo/demo.xcodeproj%22%0A)), we don't need to have `team_id`, so please recheck if we need it or not. If we need it, what is the error when running that command without team_id 🤔

Reply:
Without team id, we got this warning and the project couldn't be archived and failed with status code 65

![CleanShot 2023-04-21 at 13 09 16@2x](https://user-images.githubusercontent.com/24598204/233554145-bf283cb0-df69-4a01-8622-a0ac95ce8457.png)

Reply:
@ducbm051291 without `team_id` does not mean not setting team id. It seems you are still using `team_id` with `updateCodeSigningSettings`, please remove that parameter 🙈
```diff
updateCodeSigningSettings(
	path: Constant.projectPath,
	useAutomaticSigning: .userDefined(false),
-	teamId: .userDefined(Constant.teamId),
	targets: .userDefined([Constant.projectName]),
	buildConfigurations: .userDefined([Self.createBuildConfiguration(type: type, environment: environment)]),
	codeSignIdentity: .userDefined(type.codeSignIdentity),
	profileName: .userDefined(Self.createProfileName(type: type, environment: environment))
)
```

Reply:
@suho After removing setting teamId, the workflow failed to build app 😓

Reply:
what is the error? did you run on local?

Reply:
Same as usual, exit status code 65 

<img width="603" alt="CleanShot 2023-04-21 at 14 22 07@2x" src="https://user-images.githubusercontent.com/24598204/233569011-422179ac-2e7d-4279-860d-fc42d0249344.png">

Here are the local runs

With set teamId

<img width="615" alt="CleanShot 2023-04-21 at 14 32 56@2x" src="https://user-images.githubusercontent.com/24598204/233572878-f0c92c0c-a571-44c3-97ad-87bd4d690956.png">

Without set teamId

<img width="533" alt="CleanShot 2023-04-21 at 14 34 48@2x" src="https://user-images.githubusercontent.com/24598204/233572884-1d10d154-f5df-424c-bcf5-2d31d7e03dd3.png">

Reply:
I see, so it seems `updateCodeSigningSettings` requires teamId. Thanks for the info